src/controller/AppController.cpp

```diff
@@ -69,4 +92,35 @@ void AppController::checkForAppAndThrowIfNotFound (const juce::Identifier& appId
         throw AppUnknownException (appId);
```

Comment:
I get some build warnings on this line when building with MSVC locally

Reply:
Yes, I also saw this (like "throwing will not be caught" or similar, I'll add an issue for that).